src/components/shared/Tooltip.js

```diff
@@ -89,12 +89,14 @@ export default class Tooltip extends React.PureComponent<Props, State> {
     let offsetY = 0;
     if (interiorElement) {
       if (
-        mouseY + interiorElement.offsetHeight + MOUSE_OFFSET >
-        window.innerHeight
+        interiorElement.offsetHeight + MOUSE_OFFSET <
+        window.innerHeight - mouseY
       ) {
+        offsetY = 0;
```

Comment:
I think it should be `-MOUSE_OFFSET` here, otherwise the tooltip is too close of the mouse cursor. But please see the comment below as well.